invenio_s3/__init__.py

```diff
@@ -18,9 +18,9 @@
 
 When creating a new location which will use the S3 API, the URI needs to start
 with ``s3://``, for example
-``invenio files location S3_default s3://myserver.com/b1 --default`` will
+``invenio files location S3_default s3://my-bucket --default`` will
```

Comment:
Moderate: IIRC the name does not accept upper case (I do not understand why dashes are highlighted in the suggestions...):

```suggestion
``invenio files location s3_default s3://my-bucket --default`` will
```

Comment: As soon as this https://github.com/inveniosoftware/invenio-files-rest/pull/236 is merged, the command will be:
``` console
$ invenio files location create s3_default s3://my-bucket --default
```
Not  saying to change it now, just to be aware, for the next release.